qunetsim/objects/quantum_connection.py

```diff
@@ -0,0 +1,188 @@
+import random
+
+
+class QuantumConnection(object):
+    """
+    An object that stores quantum connection details
+    """
+
+    def __init__(self, receiver_id, model=None):
+        self._receiver_id = receiver_id
+        if model is None:
+            self._model = Fibre()  # Defaults to fibre model
```

Comment:
Default should be the identity channel

Reply:
BEC with erasure probability = 0 set as default.